docs/design-proposals/multi-tenancy/multi-tenancy-network-mizar-integration.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Arktos Multi-Tenancy Based on Mizar
```

Comment:
this design seems about network (integrating with mizar), not multi-tenancy. suggest to change the title a bit.